luvit-loader.lua

```diff
@@ -197,7 +197,7 @@ local function loader(dir, path, bundleOnly)
         return package.loaded[key]
       end
       local code = bundle.readfile(fullPath)
-      local module = loadstring(code, key)()
+      local module = loadstring(code, key)(key)
```

Comment:
I believe that change will require a few other changes as well, as the `bundle:` prefix is what is checked using debug.getinfo() in the loader. I haven't done much with the bundle side of the loader, so I'll have to look into that more and write some test cases for it.